index.js

```diff
@@ -1,15 +0,0 @@
-import express, { json } from "express";
-import cors from "cors";
-import dotenv from "dotenv";
-import authRouter from "./routes/authRouter.js";
-import movimentRouter from "./routes/movimentRouter.js";
-dotenv.config();
-
-const app = express();
-app.use(json());
-app.use(cors());
-
-app.use(authRouter);
-app.use(movimentRouter);
-
-app.listen(process.env.PORT);
```

Comment:
Aqui é legal que a porta do servidor seja uma variável de ambiente. Além de ser uma boa prática, facilita para quem for rodar seu projeto posteriormente 

```
const port = process.env.PORT || 5000;
app.listen(port , () => ...
```